src/components/table/table.component.js

```diff
@@ -21,8 +21,8 @@ let deprecatedWarnTriggered = false;
 class Table extends React.Component {
   static contextType = LocaleContext;
 
-  constructor(props) {
-    super(props);
+  constructor(props, context) {
```

Comment:
This should be in separate commit. Does this add any functionality or is it just code style?

Reply:
just code style, we can ignore it for now